subjects/make-your-game/README.md

```diff
@@ -0,0 +1,114 @@
+
+### Instructions
+
+Animation must have consistent motion, so in order to have a smooth animation (without interruptions or better named: jank animation) you must achieve a special number, [**60fps**](https://blog.algolia.com/performant-web-animations/).
+
+Performance is essential, so that's why you have to aim for less than 16.7ms(1000ms/60f), because in 16.7ms the browsers job and your work must be completed in each frame.
+
+Use [`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) to sync your changes
+
+- You can try to reuse memory to avoid jank when the garbage collector pass
+
+Jank animation can be caused by loading too much information, for instance:
+
+- **JavaScript**
+- **Styles** that considers which style apply to which element
+- **Layout** that calculates the geometry of the pages (example: recalculating the width and height of a page)
+- **Painting**, normally when layout is triggered we must repaint. Repainting an element every time it animates
+- **Compositing** that consists on placing the pages together at the end
+
+In order to improve performance we must remove the causes above, that are more costly: layout and painting.
+
+The best way to remove layout and painting is to use [`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) and [`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity)
+
+Removing layout can be done using transform:
+
+```js
+// bad
+// this will trigger the layout to recalculate everything and repaint it again
+box.style.left = `${x * 100}px`;
+
+// good
+// this way its possible to lose the layout
+box.style.transform = `translateX(${x * 100}px)`;
+```
+
+It is possible to remove painting by adding a layer:
+
+```css
+/* this will take care of the painting by creating a layer and transform it*/
+  #box {
+    width: 100px;
+    height: 100px;
+    ....
+    will-change: transform;
+  }
+```
+
+By creating a new layer you can remove painting, but "there is always a tradeoff". If we add to much layers it will increase the **composition** and **update tree**. In conclusion you must promote a new layer only if you now you are going to use it. Performance is the key to a good animation. "Performance is the art of avoiding work".
+
+### Pre-Approved List
+
+Your game will have to respect the genre of one of these games listed below. In other words, the main goal of the game has to be similar to one of these:
+
+- [Flipper/ Pinball](https://en.wikipedia.org/wiki/Pinball)
+- [Space Invaders](https://en.wikipedia.org/wiki/Space_Invaders)
+- [Donkey Kong](https://en.wikipedia.org/wiki/Donkey_Kong)
+- [Brick Breaker/ Arkanoid](https://en.wikipedia.org/wiki/Arkanoid)
+- [Pac-Man](https://pt.wikipedia.org/wiki/Pac-Man)
+- [Super Mario](https://en.wikipedia.org/wiki/Super_Mario)
+- [Tetris](https://pt.wikipedia.org/wiki/Tetris)
+- [Duck Hunt](https://en.wikipedia.org/wiki/Duck_Hunt)
```

Comment:
I would add https://en.wikipedia.org/wiki/Super_Bomberman because it's my favorite

Reply:
ok will add it to the list